src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -56,6 +64,9 @@ export interface NavBarScheduleProps {
   venueId: string;
 }
 
+const minRangeValue = 0;
+const maxRangeValue = 1;
+
```

Comment:
I've changed the logic of using those vars a bit, including `math.min` approach
but it's still needed so that we have correct range values for related events